tests/global-setup.ts

```diff
@@ -7,7 +7,8 @@ async function globalSetup() {
     process.env.FRONTEND_URL = 'not_used';
 
     console.log('Bootstrapping test database...');
-    await bootstrapDatabase();
+    const strapiInstance = await bootstrapDatabase();
+    strapiInstance.listen();
```

Comment:
Instead of starting strapi instance from playwright, we can just start listen here. Istanbul (code coverage tool) doesn't have access to code in the separately running server, but has access to code running in the main thread.